### ISSUE TITLE
[Enhancement] Per server bans.

### ISSUE THREAD

**Issue by iKaneki-Ken** [vincidev/BungeeBan#15]
Is it possible to make BungeeBan ban players on a per server basis instead of network wide? This the only thing I can think of that's preventing this to be completely full featured.

**Comment by vincidev** [comment]
Yes it is possible, but can you think of any good command for it?
Like
/sban <player> <server> <reason>

**Comment by ImNatho** [comment]
How about /ban <player> -s <~server> -m <message>
So you have tak commands like bash.
 -s single server only (Or with argument selected server only)
 -m message
 -t time (If null forever)

I might throw together a CommandUtil tonight that can parse commands in that style.

**Comment by vincidev** [comment]
Very good idea. Very possible.
